permissions/resource_permissions.go

```diff
@@ -131,15 +135,9 @@ func (a PermissionsAPI) put(objectID string, objectACL AccessControlChangeList)
 			UserName:        me.UserName,
 			PermissionLevel: "CAN_MANAGE",
```

Comment:
Is this implied permission also gone?

Reply:
yes, all admins can manage everything, except passwords and tokens, though they can manage permissions for that as well, because they're admins.

Reply:
yep, so endpoints can only be admin-created, so admins cannot loose control by design